Configurateur/front/lib/screens/landing-page/landing_page.dart

```diff
@@ -192,196 +46,138 @@ class LandingPageState extends State<LandingPage> {
-                              borderRadius: BorderRadius.circular(
-                                  20.0), // Définit le rayon du bouton arrondi
+                ),
+                const SizedBox(height: 100),
+                Container(
+                  padding: const EdgeInsets.all(20.0),
+                  child: Center(
+                      child: Row(
+                    mainAxisSize: MainAxisSize.min,
+                    children: [
+                      Column(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            Text(
+                              'Trouvez des locations selon vos \rbesoins, où vous les souhaitez',
+                              style: TextStyle(
+                                fontSize: 35,
+                                color: Provider.of<ThemeService>(context).isDark
+                                    ? darkTheme.secondaryHeaderColor
+                                    : lightTheme.secondaryHeaderColor,
+                                fontWeight: FontWeight.bold,
+                                shadows: const [
+                                  Shadow(
+                                    color: Color(0xff28666e),
+                                    offset: Offset(0.75, 0.75),
+                                    blurRadius: 1.5,
+                                  ),
+                                ],
+                              ),
+                            ),
+                            Padding(
+                              padding: const EdgeInsets.only(
+                                  top:
+                                      15), // Espacement inférieur pour le texte
+                              child: Text(
+                                'Des conteneurs disponibles partout en france !',
+                                style: TextStyle(
+                                  fontSize: 20,
+                                  color:
+                                      Provider.of<ThemeService>(context).isDark
+                                          ? darkTheme.secondaryHeaderColor
+                                          : lightTheme.secondaryHeaderColor,
+                                ),
+                              ),
                             ),
-                            textStyle: TextStyle(
-                                // fontSize: 13.0
-                                )),
-                        child: Text(
-                          'En savoir plus',
-                          style: TextStyle(
-                            color: Provider.of<ThemeService>(context).isDark
-                                ? darkTheme.primaryColor
-                                : lightTheme.primaryColor,
-                          ),
-                        ),
+                          ]),
```

Comment:
Maybe add , after the ] so that the file is more readable

Reply:
done